Description: allow nm-applet to popup wireless connections
 on Debian based systems nm-applet working in legacy mode needs to be
 called via dbus otherwise the wireless popup does not work
Author: David Mohammed <foss.freedom@gmail.com>
Forwarded: not-needed
Origin: Other
===================================================================
--- budgie-desktop-10.2.9.orig/src/session/budgie-desktop-nm-applet.desktop.in
+++ budgie-desktop-10.2.9/src/session/budgie-desktop-nm-applet.desktop.in
@@ -2,7 +2,7 @@
 Name=Network (Budgie Desktop)
 Comment=Manage your network connections
 Icon=nm-device-wireless
-Exec=nm-applet
+Exec=dbus-run-session nm-applet
 TryExec=nm-applet
 Terminal=false
 Type=Application
